src/python/WMComponent/TaskArchiver/CleanCouchPoller.py

```diff
@@ -192,10 +193,19 @@ def archiveWorkflows(self, workflows, archiveState):
         for workflowName in workflows:
             if not self.isUploadedToWMArchive(workflowName):
                 continue
+            if not self.useReqMgrForCompletionCheck:
+                # NOTE: For T0 just update the request status in central CouchDB - t0_request
+                #       and let cleanAlreadyArchivedWorkflows do the actual couchdb cleaning
+                #       after config.TaskArchiver.cleanCouchDelayHours for the current workflow
+                msg = "Setting T0 workflow: %s to status: %s at central CouchDB."
+                msg += "Local CouchDB data will be cleaned after %s hours."
+                self.logger.debug(msg, workflowName, archiveState, self.config.TaskArchiver.cleanCouchDelayHours)
```

Comment:
I think this log level could be INFO instead. Same comment for the block below.

Reply:
done

Reply:
I guess you haven't pushed this code in yet, so let me unresolve this comment.